apps/frontend/.gitignore

```diff
@@ -43,3 +43,7 @@ lerna-debug.log*
 
 # env
 .env
+
+# Playwright tests
+playwright-report
+test-results
```

Comment:
Why do we need to add this inside the frontend repo?
Also why package-lock has changes?